src/function/table/read_csv.cpp

```diff
@@ -326,6 +326,16 @@ static unique_ptr<FunctionData> CSVReaderDeserialize(Deserializer &deserializer,
 	return std::move(result);
 }
 
+vector<LogicalType> PushdownTypeToCSVScanner(ClientContext &context, optional_ptr<FunctionData> bind_data,
```

Comment:
Can we remove the return from this callback? There is only one correct return from this function that would not lead to other problems - the original types with the `new_column_types` applied over the columns. Having a return here indicates that the function has a choice in what to return which is not the case.

Reply:
Fair enough!